docs/website/docs/deployment-configurations/gpu-cuda-rocm.md

```diff
@@ -0,0 +1,203 @@
+
+=== "Nvidia/CUDA"
+
+Python packages for various IREE functionalities are regularly published
+to [PyPI][pypi]. See the [Python Bindings][python-bindings] page for more
+details. The core `iree-compiler` package includes the CUDA compiler:
+
+``` shell
+python -m pip install iree-compiler
+```
+
+!!! tip
+    `iree-translate` is installed as `/path/to/python/site-packages/iree/tools/core/iree-translate`.
+    You can find out the full path to the `site-packages` directory via the
+    `python -m site` command.
+
+=== "AMD/ROCm"
+
+Currently ROCm is **NOT supported** for the Python interface.
+
+#### Build compiler from source
+
+Please make sure you have followed the [Getting started][get-started] page
+to build IREE for Linux/Windows and the [Android cross-compilation][android-cc]
+page for Android. The CUDA compiler backend and ROCM compiler backend is compiled in by default on all
+platforms.
+
+=== "Nvidia/CUDA"
+
+Ensure that the `IREE_TARGET_BACKEND_CUDA` CMake option is `ON` when
+configuring for the host.
+
+=== "AMD/ROCM"
+
+Ensure that the `IREE_TARGET_BACKEND_ROCM` CMake option is `ON` when
+configuring for the host.
+
+## Compile and run the model
+
+With the compiler and runtime for CUDA ready, we can now compile a model
+and run it on the GPU.
+
+### Compile the model
+
+IREE compilers transform a model into its final deployable format in many
+sequential steps. A model authored with Python in an ML framework should use the
+corresponding framework's import tool to convert into a format (i.e.,
+[MLIR][mlir]) expected by main IREE compilers first.
+
+Using MobileNet v2 as an example, you can download the SavedModel with trained
+weights from [TensorFlow Hub][tf-hub-mobilenetv2] and convert it using IREE's
+[TensorFlow importer][tf-import]. Then,
+
+#### Compile using the command-line
+
+Let `iree_input.mlir` be the model's initial MLIR representation generated by
+IREE's TensorFlow importer. We can now compile them for each GPU by running the following command:
+
+=== "Nvidia/CUDA"
+
+``` shell hl_lines="3 4"
+iree/tools/iree-translate \
+    -iree-mlir-to-vm-bytecode-module \
+    -iree-hal-target-backends=cuda \
+    -iree-cuda-llvm-target-arch=<...> \
+    -iree-hal-cuda-disable-loop-nounroll-wa \
+    iree_input.mlir -o mobilenet-cuda.vmfb
+```
+
+Note that a cuda target architecture(`iree-cuda-llvm-target-arch`) of the form `sm_<arch_number>` is needed
+to compile towards each GPU architecture. If no architecture is specified then we will default to `sm_35`
+Here are a table of commonly used architecture
+
+CUDA GPU  | Target Architecture
+:--------: | :-----------:
+Nvidia K80 | `sm_35`
+Nvidia P100 | `sm_60`
+Nvidia V100 | `sm_70`
+Nvidia A100 | `sm_80`
+
+
+=== "AMD/ROCM"
+
+``` shell hl_lines="3 4"
+iree/tools/iree-translate \
+    -iree-mlir-to-vm-bytecode-module \
+    -iree-hal-target-backends=rocm \
+    -iree-rocm-target-chip=<...> \
+    -iree-rocm-link-bc=true \
+    -iree-rocm-bc-dir=<...> \
```

Comment:
highlight all rocm lines?

```suggestion
``` shell hl_lines="3 6"
iree/tools/iree-translate \
    -iree-mlir-to-vm-bytecode-module \
    -iree-hal-target-backends=rocm \
    -iree-rocm-target-chip=<...> \
    -iree-rocm-link-bc=true \
    -iree-rocm-bc-dir=<...> \
```